lib/cli/src/Cardano/CLI.hs

```diff
@@ -451,29 +454,37 @@ fullKeyEncodingDescription = mconcat
     , "- xpub1... (bech32-encoded extended public key)"
     ]
 
-xprvFromBytes :: ByteString -> Either String XPrvOrXPub
-xprvFromBytes = fmap AXPrv . left showErr . xPrvFromStrippedPubXPrvCheckRoundtrip
+aXPrvFromBytes :: ByteString -> Either String XPrvOrXPub
+aXPrvFromBytes x = maybe errLength (return . AXPrv) $ xprvFromBytes x
   where
-    showErr (ErrInputLengthMismatch expected actual) = mconcat
+    actual = BS.length x
+    expectedBytes = 96 :: Int
```

Comment:
Should probably ideally be defined in `cardano-addresses` though

Reply:
yes, `96` maybe should be explicitly exposed here : https://github.com/input-output-hk/cardano-addresses/blob/master/src/Cardano/Address/Derivation.hs#L166

Reply:
Added a todo in the code